taxi/models.py

```diff
@@ -8,7 +8,7 @@ class Manufacturer(models.Model):
     country = models.CharField(max_length=255)
 
     class Meta:
-        ordering = ["name"]
+        ordering = ["id"]
```

Comment:
Why did you change this?

Reply:
I changed this because I think that when the id in the first column is not ordered and the numbers are scattered, visually it does not look good. But if  shouldn't have done it, I'll change it back.